Source/WebKit/Shared/Cocoa/CoreIPCNSCFObject.mm

```diff
@@ -84,6 +84,20 @@
     return result;
 }
 
+bool CoreIPCNSCFObject::valueIsAllowed(IPC::Decoder& decoder, ObjectValue& value)
+{
+    // The Decoder always has a set of allowedClasses,
+    // but we only check that set when considering SecureCoding classes
+    Class objectClass;
+    WTF::switchOn(value, [&](CoreIPCSecureCoding& object) {
+        objectClass = object.objectClass();
+    }, [&](auto& object) {
```

Comment:
This should be explicitly std::nullptr_t instead of this mysterious auto&

Reply:
Note: Unlike the previous switchOn, this one is only trying to capture SecureCoding - It's intended that all other cases fall through to a null Class.